src/AppBundle/Command/AddUserCommand.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Doctrine\Common\Persistence\ObjectManager;
+use AppBundle\Entity\User;
+
+/**
+ * A command console that creates users and stores them in the database.
+ * To use this command, open a terminal window, enter into your project
+ * directory and execute the following:
+ *   $ php app/console app:add-user
+ *
+ * To output detailed information, increase the command verbosity:
+ *   $ php app/console app:add-user -vv
+ *
+ * See http://symfony.com/doc/current/cookbook/console/console_command.html
+ *
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class AddUserCommand extends ContainerAwareCommand
+{
+    const MAX_ATTEMPTS = 5;
+
+    /** @var ObjectManager */
+    private $em;
+
+    protected function configure()
+    {
+        $this
+            // a good practice is to use the 'app:' prefix to group all your custom application commands
```

Comment:
Some people get less nervous when you say "good practice" instead of "best practice".